tests/apollo/test_skvbc.py

```diff
@@ -71,7 +71,7 @@ async def test_state_transfer(self, bft_network,exchange_keys=True):
             checkpoints_num=3, # key-exchange channges the last executed seqnum
             persistency_enabled=False
         )
-        bft_network.start_replica(stale_node)
+        await bft_network.start_replica(stale_node)
```

Comment:
Hi @glevkovich I think with your latest changes, start/stop replica don't need to be async anymore.
Please correct me if I'm wrong.

Reply:
Yes, I will change it back.